pkg/mhooks/payload.go

```diff
@@ -0,0 +1,57 @@
+package mhooks
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"time"
+)
+
+func NewPayload(requestBody io.Reader) (*Payload, error) {
+	var payload Payload
+	err := json.NewDecoder(requestBody).Decode(&payload)
+	if err != nil {
+		return nil, fmt.Errorf("decoding payload: %w", err)
+	}
+
+	return &payload, nil
+}
+
+type Payload struct {
```

Comment:
I need to rename this to `Event`, will match terminology we use in our docs: https://docs.moov.io/guides/webhooks/webhook-events/#events

Reply:
renamed